### PR TITLE
	new file:   build.sh

### PR DESCRIPTION
**Pull request by hsinnan75**
new file:   meta.yaml

Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by hsinnan75** [comment]
@dpryan79 Thank you! I was invited to join Bioconda. I'd like to know how to review and merge. Do you know where I can find the documents? Thank you!

**Comment by dpryan79** [comment]
Hmm, we should probably write some documentation on that :)

The basic idea is to look over PRs and make sure that (A) they're actually building something and (B) the changes look sane. For (B), it's mostly a matter of seeing if a license is included, that the package is appropriate for bioconda (i.e., it's bio related), and that the tests at least run something meaningful and not just `ls`. When all of that looks fine, you can click on the "Review" button on the top right of the "Files changed" panel, select "Accept" and then merge people's PRs :)

**Comment by hsinnan75** [comment]
@dpryan79 Thank you for the explanation.

File: recipes/gsalign/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS} -msse4.2 -mpopcnt -fPIC" CXXFLAGS="${CXXFLAGS} -msse4.2 -mpopcnt -fPIC"
+cp bin/GSAlign bin/bwt_index $PREFIX/bin
```
Comment (mbargull) [inline recipes/gsalign/build.sh]: If `bin/bwt_index` is the same file as in gh-18694 and gh-18702, you might want to consider putting that into a separate package.
Depending on the user's `conda config` settings, having file with the same path in multiple packages will either prevent those from being installed simultaneously or lead to unexpected errors:
```
> conda config --describe path_conflict
# # path_conflict (PathConflict)
# #   The method by which conda handle's conflicting/overlapping paths
# #   during a create, install, or update operation. The value must be one
# #   of 'clobber', 'warn', or 'prevent'. The '--clobber' command-line flag
# #   or clobber configuration parameter overrides path_conflict set to
# #   'prevent'.
# # 
# path_conflict: clobber
```
The default setting is `clobber` which means `conda` will just install all packages as instructed, but will also not pay attention to the conflicting file when removing any package. Imagine the following:
```
> # install the two packages, gsalign and kart, that both contain bin/bwt_index
> conda install gsalign kart
> # remove only one of the packages
> conda remove kart
> # now, bin/bwt_index has been removed alongside all of kart
> bwt_index
bash: bwt_index: command not found
> # ergo: the currently still installed package gsalign is incomplete/broken
```

You can avoid this conflict easily if you put `bin/bwt_index` in a separate package and add that package as a `run` dependencies to the other packages.

**Pull request merged**